format_strawberryfield.libraries.yml

```
@@ -13,11 +13,17 @@ iiif_openseadragon_strawberry:
   version: 1.0
   js:
     js/iiif-openseadragon_strawberry.js: {minified: false}
+    js/jquery_dollar.js: {preprocess: false, minified: false, weight: -10}
   dependencies:
     - core/jquery
+    - core/jquery.once
     - core/drupal
     - core/drupalSettings
+    - format_strawberryfield/iiif_openseadragon
     - format_strawberryfield/annotoriousopenseadragon
```

```suggestion
    - format_strawberryfield/annotoriousopenseadragon
    - format_strawberryfield/opencv
```
Adding opencv as a dependency seems to get rid of the `FS error` in my testing.